client/pages/index.tsx

```diff
@@ -1,10 +1,24 @@
-import styles from "../styles/index.module.css"
-import LandingHeader from "../components/LandingHeader"
+import styles from "../styles/index.module.css";
+import LandingHeader from "../components/LandingHeader";
+import FEAuth from "../components/FEAuth";
+import React, { useState } from "react";
 
-const HomePage = () => (
-    <div className={styles.headerContainer}>
-        <LandingHeader />
-    </div>
-);
+const HomePage = () => {
+    const [display, setDisplay] = useState(true);
+    const landingContainer = display ? styles.displayFlex : styles.displayNone;
+    const authContainer = display ? styles.displayNone : styles.displayFlex;
+    return (
+        <div >
+            <div className={landingContainer}>
+                <LandingHeader 
+                onClickHost={()=>setDisplay(!display)} 
+                onClickJoin={()=>console.log("some Join Stuff")}/>
+            </div>
+            <div className={authContainer}>
+                <FEAuth />
+            </div>
+        </div>
+    );
```

Comment:
Probably a lot simpler to output only what is necessary instead of hiding it with CSS